_maps/RandomRuins/IceRuins/icemoon_underground_hermit.dmm

```diff
@@ -0,0 +1,452 @@
+"bz" = (
+/obj/item/flashlight/lantern,
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"cI" = (
+/turf/open/floor/plating,
+/area/ruin/powered/shuttle)
+"de" = (
+/obj/item/flashlight/lantern,
+/turf/open/floor/plating/asteroid/snow/icemoon,
+/area/icemoon/underground/explored)
+"ec" = (
+/obj/item/clothing/suit/hooded/wintercoat,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"ei" = (
+/obj/machinery/hydroponics/soil,
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"gr" = (
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"ha" = (
+/obj/machinery/light{
+	dir = 8
+	},
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"hv" = (
+/obj/structure/bonfire/prelit,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"hN" = (
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"ii" = (
+/obj/effect/mob_spawn/human/hermit{
+	flavour_text = "You've been living on the ice moon for at least 40 years now! You've survived the ice whelps, hunted those damn bears and tamed a few wolves. What do those outsiders want?";
+	mob_name = "a grumy old man";
+	name = "Cryostasis bed";
+	short_desc = "You've been hunting polar bears for 40 years now! What do these 'NaniteTrans' newcomers want?"
+	},
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"lH" = (
+/obj/structure/barricade/wooden/crude/snow,
+/turf/open/floor/plating/asteroid/snow/icemoon,
+/area/icemoon/underground/explored)
+"mN" = (
+/turf/closed/wall/mineral/wood,
+/area/ruin/powered/shuttle)
+"oJ" = (
+/obj/structure/sink,
+/turf/open/floor/plating,
+/area/ruin/powered/shuttle)
+"pI" = (
+/obj/item/chair/wood/wings,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"sC" = (
+/obj/item/gun/ballistic/rifle/boltaction,
+/obj/structure/table/wood,
+/obj/item/flashlight/lantern,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"sM" = (
+/obj/structure/chair/comfy/beige,
+/obj/machinery/light{
+	dir = 8
+	},
+/obj/machinery/light{
+	dir = 1
+	},
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"wH" = (
+/obj/item/tank/internals/emergency_oxygen/engi,
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"EV" = (
+/obj/item/storage/bag/plants/portaseeder,
+/obj/item/storage/firstaid/medical,
+/obj/item/storage/bag/ore,
+/obj/structure/table/wood,
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"GU" = (
+/obj/item/storage/toolbox/emergency,
+/turf/open/floor/plating,
+/area/ruin/powered/shuttle)
+"II" = (
+/obj/item/paper/guides/jobs/hydroponics,
+/turf/open/floor/plating,
+/area/ruin/powered/shuttle)
+"JI" = (
+/obj/machinery/door/airlock/wood{
+	name = "Frozen Shack"
+	},
+/obj/structure/fans/tiny/invisible,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"Oe" = (
+/obj/item/seeds/plump,
+/obj/item/seeds/plump,
+/obj/item/seeds/tower,
+/obj/item/seeds/reishi,
+/obj/item/reagent_containers/food/snacks/grown/mushroom/glowshroom/glowcap,
```

Comment:
Maybe glowshrooms instead? Or is it intended as a way to recharge things?

Reply:
Oop, that's a mistake. I'll fix that.